khal/controllers.py

```diff
@@ -21,6 +21,8 @@
 # WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #
 
+from __future__ import unicode_literals
+
```

Comment:
Probably khal doesn't even use any APIs where this matters, I just wouldn't do it as a general rule.